src/test/java/ladder/domain/HeightTest.java

```diff
@@ -0,0 +1,32 @@
+package ladder.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class HeightTest {
+
+    @Test
+    public void 세로길이_객체를_생성할_수_있다(){
```

Comment:
```suggestion
    public void 세로길이_객체를_생성할_수_있다() {
```

매의 눈 @_@

또한 jUnit5에서는 @DisplayName을 이용하여 의미를 전달하실 수 있습니다.

만약 한글로 메서드명을 짓는것을 의도하였다면 클래스에 @SuppressWarnings("NonAsciiCharacters")를 적용하여 IDE의 경고를 제거해보시는게 어떨까요?

Reply:
@DisplayName 은 알고 있었지만, 한글 메서드명으로 대체할 수 있다 생각하여 쓰지 않고 있었습니다 😂
 @SuppressWarnings("NonAsciiCharacters") 는 처음 알았네요! 꿀팁 전수 감사합니다 👍
(컨벤션은 고쳐두도록 하곘습니다 ㅎㅎ)